content/departments/cloud/roles-and-responsibilities.md

```diff
@@ -0,0 +1,106 @@
+# Cloud Team Roles and Responsibilities
+
+The aim of this document is to identify the roles within the Cloud team and align on the responsibilities for each role.
+
+## Roles
+
+- [Cloud Engineer](#cloud-engineer)
+- [Cloud EM and PM](#cloud-em-and-pm)
+- [DRI](#dri)
+- [On-call Engineer](#on-call-engineer)
+- [Security Engineer](#security-engineer)
+
+### Cloud Engineer
+
+- Make continuous progress on roadmap items
+- Implementation work
+- Push for architecture/feature designs that work the best for the cloud
+- Regularly team the team on their work, encountered problems, blockers, and delays
+- Highlight potential security issues for review
+- Complete work as defined by DRIs
+
+- Documentation and testing
+- Ensure the team stays technically flexible to deliver on changing requirements
+- Identify areas of technical improvement, process improvement, and paring down tech debt
+
+- Technical decision-making
+- Code reviews of Go code and IaaC languages like Terraform
+- Identify and escalate gaps in processes, tooling, and ownership that affect Cloud roadmap
+- Identify team weaknesses and define processes for improvement
+- Maintain a broad view on state and status of team initiatives and roadmap items
+- Grow and maintain knowledge of GCP resources and when to use managed resources vs self-hosting
```

Comment:
The rendered result doesn't seem to do what we intend, every item has much larger line height without any grouping:

<img width="574" alt="CleanShot 2022-10-24 at 11 11 30@2x" src="https://user-images.githubusercontent.com/2946214/197441401-9c2325b3-88ac-4705-b1c9-83604a28d1f7.png">

So we should either put them together or give them subgroup titles:

```suggestion
- Make continuous progress on roadmap items
- Implementation work
- Push for architecture/feature designs that work best for the cloud
- Regularly team the team on their work, encountered problems, blockers, and delays
- Highlight potential security issues for review
- Complete work as defined by DRIs
- Documentation and testing
- Ensure the team stays technically flexible to deliver on changing requirements
- Identify areas of technical improvement, process improvement, and paring down tech debt
- Technical decision-making
- Code reviews of Go code and IaaC languages like Terraform
- Identify and escalate gaps in processes, tooling, and ownership that affect Cloud roadmap
- Identify team weaknesses and define processes for improvement
- Maintain a broad view on state and status of team initiatives and roadmap items
- Grow and maintain knowledge of GCP resources and when to use managed resources vs self-hosting
```

Reply:
Same for the other lists.